.github/workflows/docs/intro.txt

```diff
@@ -1,8 +1,8 @@
 pytket-extensions
 =================
 
-.. image:: CQCLogo.png
-   :width: 120px
+.. image:: Quantinuum_logo.png
+   :width: 200px
    :align: right
 
 
```

Comment:
Do you want to replace the `.. _CQC: https://cambridgequantum.com` link, too?

Reply:
Thanks for spotting that - I've replaced it.

Reply:
Perfect.